server/features/EmailInbox/EmailInbox.ts

```diff
@@ -20,14 +21,15 @@ export async function configureEmailInboxes(): Promise<void> {
 		active: true,
 	});
 
+	logger.info('Clearing old email inbox registrations');
 	for (const { imap } of inboxes.values()) {
 		imap.stop();
 	}
 
 	inboxes.clear();
 
 	for await (const emailInboxRecord of emailInboxesCursor) {
-		console.log('Setting up email interceptor for', emailInboxRecord.email);
+		logger.info('Setting up email interceptor for', emailInboxRecord.email);
```

Comment:
this usage will be deprecated (we need to add a warning for that)

please log either an object or use placeholders: https://github.com/pinojs/pino/blob/master/docs/api.md#message-string